<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NomenclatureReportDialog</class>
 <widget class="QDialog" name="NomenclatureReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblMonth">
     <property name="text">
      <string>Месяц</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblReceiverPerson">
     <property name="text">
      <string>Ответственный</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="COrgComboBoxEx" name="cmbSupplierOrg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbReceiver">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финанисирования</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblClientId">
     <property name="text">
      <string>Код пациента</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTimeEdit" name="edtEndTime"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblSupplierOrg">
     <property name="text">
      <string>Внешний поставщик</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbSupplier">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLineEdit" name="edtClientId"/>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lblSigna">
     <property name="text">
      <string>Способ применения</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateEdit" name="edtMonth">
     <property name="displayFormat">
      <string>yyyy.MM</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblNomenclature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Но&amp;менклатура</string>
     </property>
     <property name="buddy">
      <cstring>cmbNomenclature</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QLineEdit" name="edtSigna"/>
   </item>
   <item row="2" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="CNomenclatureComboBox" name="cmbNomenclature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="3">
    <widget class="CPersonComboBoxEx" name="cmbReceiverPerson"/>
   </item>
   <item row="6" column="1">
    <widget class="CDateEdit" name="edtEndDateOption">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTimeEdit" name="edtBegTime"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblReceiver">
     <property name="text">
      <string>Получатель</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblSupplier">
     <property name="text">
      <string>Поставщик</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="edtMonthOption">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy.MM</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkMonth">
     <property name="text">
      <string>Месяц</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="5" column="1">
    <widget class="CDateEdit" name="edtBegDateOption">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblDatePeriod">
     <property name="text">
      <string>Период по дате</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="cmbDatePeriod">
     <item>
      <property name="text">
       <string>начала госпитализации</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>окончания госпитализации</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbNomenclatureKind"/>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbNomenclatureType"/>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbNomenclatureClass"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblNomenclatureKind">
     <property name="text">
      <string>Номенклатура Вид</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblNomenclatureClass">
     <property name="text">
      <string>Номенклатура Класс</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblNomenclatureType">
     <property name="text">
      <string>Номенклатура Тип</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1" colspan="3">
    <widget class="QCheckBox" name="chkCountingTotal">
     <property name="text">
      <string>Подсчет ИТОГО</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CNomenclatureComboBox</class>
   <extends>QComboBox</extends>
   <header>Stock/NomenclatureComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtMonth</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtBegTime</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtEndTime</tabstop>
  <tabstop>chkMonth</tabstop>
  <tabstop>edtMonthOption</tabstop>
  <tabstop>chkBegDate</tabstop>
  <tabstop>edtBegDateOption</tabstop>
  <tabstop>chkEndDate</tabstop>
  <tabstop>edtEndDateOption</tabstop>
  <tabstop>edtClientId</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbNomenclature</tabstop>
  <tabstop>cmbNomenclatureClass</tabstop>
  <tabstop>cmbNomenclatureKind</tabstop>
  <tabstop>cmbNomenclatureType</tabstop>
  <tabstop>edtSigna</tabstop>
  <tabstop>cmbSupplier</tabstop>
  <tabstop>cmbSupplierOrg</tabstop>
  <tabstop>cmbReceiver</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkCountingTotal</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>cmbReceiverPerson</tabstop>
  <tabstop>cmbDatePeriod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NomenclatureReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NomenclatureReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonth</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtMonthOption</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBegDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegDateOption</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEndDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndDateOption</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonth</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkBegDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonth</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkEndDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBegDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkMonth</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBegDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkEndDate</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
